packages/instrumentation-openai/src/instrumentation.ts

```diff
@@ -520,4 +545,49 @@ export class OpenAIInstrumentation extends InstrumentationBase<any> {
       ? this._config.traceContent
       : true;
   }
+
+  private _addLogProbsEvent(
+    span: Span,
+    logprobs:
+      | ChatCompletion.Choice.Logprobs
+      | ChatCompletionChunk.Choice.Logprobs
+      | CompletionChoice.Logprobs
+      | null,
+  ) {
+    let result: { token: string; logprob: number }[] = [];
+
+    if (!logprobs) {
+      return;
+    }
+
+    if ("content" in logprobs) {
+      if (logprobs.content) {
```

Comment:
Isn't it easier to do `(logprobs as ChatCompletion.Choice.Logprobs).content` and then you can do it in a single `if`?